updated images

McFe opened the pull request:
Updated powerup, leaderboard and market images to be current, also changed header font color to be the same as/more similar to powerup

McFe committed:
updated images

Updated powerup, leaderboard and market images to be current, also changed header font color to be the same as/more similar to powerup

WeslayCodes commented on main.css:
```diff
@@ -42,12 +42,12 @@ html, body {
 
 h1 {
-    color: #51d181;
+    color: #40DEA9;
```
Colors aren't meant to be powerup colors, they're supposed to match the green of Boar Central.

McFe committed:
fix for updated images

changed the css colors back, also replaced new TS powerup image with JE powerup image

McFe committed:
changed image sizes

changed the image sizes to be closer to the old images
